sycl/test-e2e/ESIMD/matrix_transpose.cpp

```diff
@@ -5,11 +5,9 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// FIXME: Investigate Windows-specific failures
```

Comment:
Could you keep it disabled on Windows until we enable the validation on Windows, please?
@uditagarwal97, FYI.